content/influxdb/v2.0/reference/flux/stdlib/schema/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Flux InfluxDB schema package
+list_title: InfluxDB schema package
+description: >
+  The Flux InfluxDB schema package provides functions for managing data from an InfluxDB v1.x
+  database or structured using the InfluxDB schema data structure.
+  Import the `influxdata/influxdb/schema` package.
+aliases:
+  - /influxdb/v2.0/reference/flux/functions/influxdb-v1/
+menu:
+  influxdb_2_0_ref:
+    name: InfluxDB Schema
+    parent: Flux standard library
+weight: 202
+influxdb/v2.0/tags: [functions, influxdb-v1, package]
+---
+
+InfluxDB schema Flux functions provide tools for managing data from an InfluxDB v1.x
+database or structured using the InfluxDB v1 data structure.
+Import the `influxdata/influxdb/schema` package:
```

Comment:
We should remove the v1 language here, these schema functions apply equally to v1 or v2, hence why we have moved them.